eoscostarica/guide.eoscostarica.io
# refactor(frontend): change the format of images from png to webp

### Change the format of images

### What does this PR do?

- resolves #297 
- Add webp formatted images
- Add alternative text to some images

### Steps to test
1. Check that the references of the static images correspond to the webp format
1. Check that the added alternative text is correct

#### CheckList
